app/controllers/api/users.php

```diff
@@ -438,6 +439,45 @@
         $response->dynamic($user, Response::MODEL_USER);
     });
 
+App::patch('/v1/users/:userId/verification')
```

Comment:
Isn't this conflicting with email verification? 
```suggestion
App::patch('/v1/users/:userId/verification/phone')
```